vignettes/Data_processing.Rmd

```diff
@@ -1496,6 +1477,43 @@ fullPlot
 
 **Figure 4:** This figure illustrates the results from the Level 5 analyses in the <font face="CMTT10"> tcpl </font> pipeline package including the best model fit and subsequent point-of-departure (POD) estimates. The light-blue shaded region represents the estimated efficacy cutoff ($\mathit{coff}$). Each of the concentration-response models fit in Level 4 are included in the plot, where the blue curve indicates the best model fit for the observed data (white circles) and the rest are depicted by the gray curves.  The horizontal lines show the activity responses from which potency estimates of interest are defined, and the vertical lines show the corresponding POD estimates.  The black point shows the AC~5~ (concentration producing $5 \%$ of the maximal response), the purple point shows the AC~10~ (concentration producing $10 \%$ of the maximal response), the yellow point shows the BMD (benchmark dose), the orange point shows the ACC (concentration producing a response at the efficacy cutoff), and the pink point shows the AC~50~ (concentration producing $50 \%$ of the maximal response).
 
+#### Fit Categories
+
+<center>![<font style="font-size:15px"><i>Fit Category Tree</i></font>](img/Fig5_mc5_fitc.png)</center>
+
+**Figure 5:** All concentration series enter the fit category ($\mathit{fitc}$) hierarchical tree and move through activity, efficacy,
+and potency comparisons to assign final $\mathit{fitc}$ (indicated by gray shading). For continuity purposes, $\mathit{fitc}$
+numbering has been conserved from past <font face="CMTT10">tcpl</font> versions. $\mathit{Conc}$ = concentration; $\mathit{hitc}$ = hit call; $\mathit{|top|}$ =
+absolute value of the modeled curve top; $\mathit{coff}$ = cutoff; $\mathit{log_c(min)}$ = minimum log~10~ concentration tested;
+$\mathit{log_c(max)}$ = maximum log~10~ concentration tested; AC~50~ = $50 \%$ activity concentration; AC~95~ = $95 \%$ activity
+concentration.
+
+Fit category ($\mathit{fitc}$) is largely based upon the relative efficacy and, in the case of actives, the location of the AC~50~ and
+concentration at $95 \%$ activity (an estimate of maximum activity concentration, AC~95~) compared to the
+tested concentration range. Figure 5 illustrates the $\mathit{fitc}$ hierarchical tree used to determine the $\mathit{fitc}$. All
```

Comment:
Remove this sentence "Figure 5 illustrates the fitc hierarchical tree used to determine the fitc."